app/Jobs/LaunchSearchRecipeJob.php

```
@@ -33,21 +29,39 @@
+                'error' => $e->getMessage(),
+                'trace' => $e->getTraceAsString(),
+            ]);
+
+            throw $e;
+        } catch (Throwable $e) {
+            Log::error('❌ Unexpected error while processing SearchRecipe.', [
+                'error' => $e->getMessage(),
+                'trace' => $e->getTraceAsString(),
+            ]);
+
+            throw new SearchRecipeException('An unexpected error occurred while processing SearchRecipe.');
```

PHPStan (2.0.3)

Instantiated class App\Jobs\SearchRecipeException not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)